.github/workflows/tag-and-release.yaml

```
@@ -0,0 +1,32 @@
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm ci
+      - name: Build
+        run: npm run package
+      - name: Test Local Action
+        id: test-local-action
+        uses: ./
```

actionlint

file "dist/index.js" does not exist in "/inmem/34/1a52842d-81a5-4686-a579-36f7599ca60a/home/jailuser/git". it is specified at "main" key in "runs" section in "Release Automation Action" action

(action)